draft-packages/menu/package.json

```diff
@@ -33,6 +33,7 @@
   "license": "MIT",
   "dependencies": {
     "@kaizen/component-library": "^12.4.5",
+    "@kaizen/button": "^1.0.4",
     "@kaizen/draft-button": "^5.6.5",
```

Comment:
Missing a comment for this one? 🤔

Reply:
No I think that one just should have a draft button, will remove it 👍